### DISCUSSION TITLE
Same zone across two cameras triggers both cameras?

### DISCUSSION THREAD

**Discussion by ediguidedog** [blakeblackshear/frigate#3478] [Ask A Question] [answered]
I've had a play and cant get it to work but I'm not sure if that's because it's not meant to work that way and I'm reading something into the docs that isn't there or it should work and I'm doing it wrong 

So suppose I have two cameras (cam1 and cam2) that both cover differing parts of the same area of interest with no actual overlap. Let's call it the 'front path' . If I create a zone on each camera that covers their own respective part of the path and name both zones 'front path' and make it a required zone for each camera should someone triggering an event by entering this 'front path' zone of cam1 automatically trigger an event on cam2 despite them not being in the cam2 'front path' zone?

I guess what I'm asking is there a way, using just config, that an event on one camera can trigger an event on another?

**Comment by NickM-27** [comment, marked as answer]
I don't think it works this way today, and there's not a way to do that. 

Using the same name (I believe) works the other way in that if both cameras have some overlap and the same zone name and sees one person, the MQTT sensors will just report 1. 

I'll have to check if there's an existing feature request for that.

**Reply by ediguidedog** [reply]
Thanks.  I'm not really sure how useful a feature it would be and I was really just trying it out because the docs mentioned you could have the same zone name across multiple cameras and I thought this might be the reason. I guess if you had cameras covering multiple entrances to the same space it might be useful to trigger as a group in order to maximize the possibility of capturing the face of the person triggering. I suppose it comes down to how easy it would be to implement.  If it was implemented it gives another configuration option with no loss of existing functionality. 

As an aside I have to say how impressed I've been with frigate. As a long time ZoneMinder user I thought I'd give frigate a whirl and it didn't take long to make the switch permanent. Even with CPU rendering while I wait for my coral to arrive it's streets ahead and my false alarms are practically zero.  Great job and thanks to all involved.
